.github/actions/e2e-tests/action.yml

```diff
@@ -40,7 +40,6 @@ runs:
         SLACK_TOKEN: ${{ inputs.slack_token }}
         STATIC_ASSET_CDN_URL: /
         ENV: ${{ inputs.environment }}
-        ENABLE_INFLUXDB_FEATURES: ${{ inputs.enable_influxdb_features }}
```

Comment:
So, I guess we're removing this from here as the environment variable is no longer used? We rely on the DISABLE_ANALYTICS_FEATURES from the API, is that right? 

If so, we should remove the input from the list of `inputs` above and remove the value wherever we're calling this action.

Reply:
Great catch, fixed. Also fixed the docs.

Reply:
The following workflows do use this input:

- `.github/workflows/platform-pull-request.yml`
- `.github/workflows/platform-test-merge-to-main.yml`

Or am I missing something here?

Reply:
You're right, it's meaningless now. Removed both. the input and the references.

Reply:
Agree, let's control this from the backend. Moved `disable_analytics_features` input to `run-local-api`.